test/test_core.py

```diff
@@ -2757,6 +2757,21 @@ def test_copyop(self):
     # (llvm-gcc copies items one by one).
     self.do_core_test('test_copyop.cpp')
 
+  # Verify that when passing -mno-bulk-memory -mno-bulk-memory-opt, that the
+  # resulting .wasm module should not have the bulk-memory Data Count section.
+  @no_wasm2js('This test verifies .wasm module behavior')
```

Comment:
There is an existing `test_wasm_features` test that is supposed to code this kind of thing.  Maybe this can be added to that existing test?  Or at least co-located with it and named similarly to it?

Reply:
I placed this in the core test suite, because it looks like this is optimization flag dependent. I.e. it does not fail in -O3, but fails in -Os and -O3. So testing different optimization modes seems in order.

Reply:
Maybe `test_wasm_features` should be in core for the same reasons then?